modules/steam/hm.nix

```diff
@@ -0,0 +1,60 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.stylix.targets.steam;
+
+  adwaitaCustomCss = config.lib.stylix.colors {
+    template = ./custom.mustache;
+    extension = "css";
+  };
+
+  adwaitaTheme = pkgs.stdenv.mkDerivation (_self: {
+    name = "Adwaita-for-Steam";
+    version = "3.1";
+
+    src = config.stylix.inputs.adwaita-for-steam;
+
+    buildInputs = with pkgs; [ python3 ];
+
+    buildPhase = ''
+      mkdir --parents $out/steamui/css
+      touch $out/steamui/css/library.css
+      cp ${adwaitaCustomCss} ./custom/custom.css
+      python3 install.py --custom-css --target $out
+    '';
+  });
+in
+{
+  options.stylix.targets.steam = {
+    enable = config.lib.stylix.mkEnableTarget "Steam" true;
+    adwaitaForSteam.enable = config.lib.stylix.mkEnableTarget "Adwaita for Steam" true;
+  };
+
+  config =
+    lib.mkIf (config.stylix.enable && cfg.enable && cfg.adwaitaForSteam.enable)
+      {
+        home.activation.backupSteamCss =
+          let
+            shellScript = pkgs.writeShellScript "backup-steam-css" ''
+              css_dir="${config.xdg.dataHome}/Steam/steamui/css"
```

Comment:
This assumes that `xdg.dataHome` contains no `"` character, which is probably safe to assume:

> ```
> xdg.dataHome
>     Absolute path to directory holding application data.
>
>     Sets ‘XDG_DATA_HOME’ for the user if ‘xdg.enable’ is set ‘true’.
>
>     Type: absolute path
>
>     Default: "~/.local/share"
>
>     Declared by:
>         <home-manager/modules/misc/xdg.nix>
> ```
>
> -- home-configuration.nix(5)